tests/WE2E/test_configs/grids_extrn_mdls_suites_community/config.grid_RRFS_CONUScompact_25km_ics_RRFS_lbcs_RRFS_suite_RRFS_v1beta.yaml

```diff
@@ -0,0 +1,34 @@
+metadata:
+  description: |-
+    This test is to ensure that the workflow running in community mode
+    completes successfully on the RRFS_CONUScompact_25km grid using the RRFS_v1beta
+    physics suite. It uses RRFS forecasts mapped onto 3-km regular grid (rrfs*.conus.grib2) for
+    ICs and LBCs. This test uses old v1 sfc_data, not the v2 fractional grid sfc_data.
+user:
+  RUN_ENVIR: community
+platform:
+  EXTRN_MDL_DATA_STORES: aws
+workflow:
+  CCPP_PHYS_SUITE: FV3_RRFS_v1beta
+  PREDEF_GRID_NAME: RRFS_CONUScompact_25km
+  DATE_FIRST_CYCL: '2024060517'
+  DATE_LAST_CYCL: '2024060517'
```

Comment:
It looks like the data available on https://noaa-rrfs-pds.s3.amazonaws.com/ has been modified.  The 20240605 case only has 00, 06, 12, and 18 hour cycles available.  The new WE2E is now failing because the necessary data is no longer available for the 17z cycle.  If the data has been staged on machines, then it would be best to use the staged data.  Otherwise, please update the `DATE_FIRST_CYCL` and `DATE_LAST_CYCL` to use 00, 06, 12, or 18 hour cycles.

Reply:
The changes must have happened since Friday... There are a couple of options - 
- Keep the same date and use staged data for this case (it's already staged; only test configuration file needs to be reversed to earlier version of using standard data locations)
- Use a different date/time, i.e. 1800 UTC

Due these changes in data availability, documentation also needs tweaking in directions to downloading RRFS data. I will include these corrections in the next commit

Reply:
Since the data is already staged and we already have a WE2E test for pulling data from AWS, I think it is fine to update the test configuration file to use the staged data.

Reply:
Updated the RRFS ICS/LBCS test with using the staged data.